exercises/practice/resistor-color-duo/.docs/instructions.md

```diff
@@ -29,5 +29,7 @@ The band colors are encoded as follows:
 - White: 9
 
 From the example above:
-brown-green should return 15
-brown-green-violet should return 15 too, ignoring the third color.
+
+`['brown', 'green']` should return `15`
+
+`['brown', 'green', 'violet']` should return `15` too, ignoring the third color.
```

Comment:
```suggestion
`['brown', 'green', 'violet']` should return `15`, ignoring the third color.
```

Could be helpful to remove the ambiguity of _too_ so that there is no reliance on knowing the line above?

Reply:
@pertrai1 I will mark this as resolved, as explained below, instructions should not be changed in this repo.

Reply:
Thank you, Rob and June for your feedback on this PR!